index.html

```diff
@@ -0,0 +1,116 @@
+            </div>
+      </section>
+    </div>
+    
+        <section id="main-program" class="main-program">
+          <h2 class="main-title">Main Program</h2>
+          <div class="orange-line"></div>
+            <div class="main-items">
+              <div class="cards">
+                <img src="./images/intro-icon.png" alt="">
+                <h3 class="orange-main">Intro</h3>
+                  <p>Listen to the pro players from various countries about the game.</p>
+              </div>
+                <div class="cards">
+                    <img src="./images/1v1.png" alt="">
+                  <h3 class="orange-main">1VS1</h3>
+                  <p>Watch 1vs1 from the pro players and learn about their mechanics.</p>
+                </div>
+                  <div class="cards">
+                    <img src="./images/q&a.png" alt="">
+                    <h3 class="orange-main">Q&A</h3>
+                    <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+                  </div>
+                    <div class="cards">
+                        <img src="./images/play.png" alt="">
+                      <h3 class="orange-main">Play with us!</h3>
+                      <p>Join the draw to be able to play and learn with a professional Apex player.</p>
+                    </div>
+                      <div class="cards">
+                        <img src="./images/community.png" alt="">
+                        <h3 class="orange-main">Apex community</h3>
+                        <p>Join the Apex Legends community to share your thoughts with other players and learn all together.</p>
+                      </div>
+            </div>
+                <button class="join-button">Join Apex Legends Webinar 2022</button>
+                <a class="see-program" href="">SEE THE WHOLE PROGRAM</a>
+        </section>
+
+        <section id="speakers" class="speakers">
+          <h2 class="speakers-title">Featured Speakers</h2>
+            <div class="orange-line"></div>            
+              <div id="speakersjs" class="speakers-cont"></div>
+              <button class="more-button">
+                MORE
+              </button>
+        </section>    
+
+        <section class="partner-section">
+          <div class="partner-title">
+          <h3>Partner</h3>
+          <div class="partner-line"></div>
+        </div>
+        <div class="partner-images">
+          <img class="corsair" src="./images/corsair.png" alt="">
+          <img class="logi" src="./images/logi.png" alt="">
+          <img class="razer" src="./images/razer.svg" alt="">   
+        </div>     
+        </section>
+  
+        <footer class="footer-index">
+          <img src="./images/apex-logo1.png" alt="apex-logo">
+          <p>2022 Apex Legends USA. Some rights reserved.</p>
+        </footer>
```

Comment:
- [ ] I'm unable to see the footer and partner  even see more not working please it would be better to see the footer and partner as in the design

Reply:
Thank you for the review, but I already made the required changes to the footer and partner section, I did it as the last reviewer told me to do, in the homepage mobile version you will not be able to see any partner or footer section because that shouldn't be there, and when you change to the about page, you will see that it is there, I did it just like the template.

Reply:
@youmari

Reply:
@Goruchie from my side wasn't working the more button and also I wasn't able to see the footer but I think you solved right now